hbase-oss/src/main/java/org/apache/hadoop/hbase/oss/Constants.java

```diff
@@ -18,9 +18,11 @@
 
 package org.apache.hadoop.hbase.oss;
 
-import org.apache.hadoop.classification.InterfaceAudience;
-import org.apache.hadoop.classification.InterfaceStability;
+import org.apache.yetus.audience.InterfaceAudience;
+import org.apache.yetus.audience.InterfaceStability;
 
+@InterfaceAudience.LimitedPrivate({"HBase"})
+@InterfaceStability.Unstable
 public class Constants  {
   public static final String DATA_URI = "fs.hboss.data.uri";
   public static final String SYNC_IMPL = "fs.hboss.sync.impl";
```

Comment:
Should we rename these properties to make it consistent with current package names (replace hboss by hbase.oss, for instance).

Reply:
Actually never thought of it as being correlated with the package name, more fs.<scheme>. being the standard config format for all FS's, even several which don't have their code living in packages named like that. But although we could do something like this in theory, no one is going to use hboss:// URIs (or even set fs.hboss.data.uri in practice). I think DATA_URI should at least be more obviously just for test configs. I would say that the others should just follow HBase's config convention, and hbase.oss seems consistent with that. My only hesitation is if this module becomes generally useful to other Hadoop clients. One of the pros of not doing this in S3A is that we don't have to solve the general case, we can focus just on HBase's needs, but a very general locking approach seems to have been doing well, so... maybe this will be more generally interesting? And hbase.* configs won't match that well. </overthinking>

Reply:
let's just leave it like this. if we get to removing the "alpha" label we can work out if the property name is an issue.